src/redux/polygonExplorer.js

```diff
@@ -0,0 +1,404 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { Contract } from '@ethersproject/contracts';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { toLower } from 'lodash';
+import {
+  COVALENT_ANDROID_API_KEY,
+  COVALENT_IOS_API_KEY,
+} from 'react-native-dotenv';
+import { polygonEnabled } from '../config/debug';
+// eslint-disable-next-line import/no-cycle
+import { addressAssetsReceived, fetchAssetPrices } from './data';
+// eslint-disable-next-line import/no-cycle
+import { emitAssetRequest, emitChartsRequest } from './explorer';
+import { AssetTypes } from '@rainbow-me/entities';
+//import networkInfo from '@rainbow-me/helpers/networkInfo';
+import networkInfo from '@rainbow-me/helpers/networkInfo';
+import networkTypes from '@rainbow-me/helpers/networkTypes';
+import {
+  balanceCheckerContractAbi,
+  chainAssets,
+  MATIC_MAINNET_ADDRESS,
+  MATIC_POLYGON_ADDRESS,
+  POLYGON_MAINNET_RPC_ENDPOINT,
+  WETH_ADDRESS,
+} from '@rainbow-me/references';
+import { ethereumUtils } from '@rainbow-me/utils';
+import logger from 'logger';
+
+// -- Constants --------------------------------------- //
+const POLYGON_EXPLORER_CLEAR_STATE = 'explorer/POLYGON_EXPLORER_CLEAR_STATE';
+const POLYGON_EXPLORER_SET_ASSETS = 'explorer/POLYGON_EXPLORER_SET_ASSETS';
```

Comment:
This action `SET_ASSETS` is not used (same in the optimism and arbitrum explorer)